apps/playground/src/app/modules/components/toggle-module/toggle-module.component.html

```diff
@@ -0,0 +1,176 @@
+<section class="module-description">
```

Comment:
The new version :
```html
<section class="module-description">
  <h2>DsfrToggleComponent</h2>
  <p>
    The <code>DsfrToggleComponent</code> purpose is to represent a toggle
    element.
  </p>
  <p>
    It is based on the
    <a
      target="_blank"
      hreflang="fr"
      title="The documentation page for the Toggle Component in the DSFR"
      href="https://www.systeme-de-design.gouv.fr/elements-d-interface/composants/interrupteur">
      Toggle
      <mat-icon>open_in_new</mat-icon>
    </a>
    component in the
    <a
      target="_blank"
      hreflang="fr"
      title="The DSFR documentation"
      href="https://www.systeme-de-design.gouv.fr/">
      DSFR
      <mat-icon>open_in_new</mat-icon>
    </a>
    .
  </p>
  <p>
    It has several <code>@Input()</code> properties and one
    <code>@Output()</code> property:
  </p>

  <dl class="input-properties">
    <play-input-property-definition
      name="toggleId"
      [required]="true"
      type="It takes a string">
      It is required and given to the <code>id</code> attribute of the
      underlying <code>input</code> tag. It is also used to generate the
      <code>aria-describedby</code> attribute
    </play-input-property-definition>
    <play-input-property-definition
      name="label"
      [required]="true"
      type="It takes a string">
      The label of the toggle. It is mandatory for accessibility reasons. You
      can, however, choose not to display it if it suits your use-case. Keep in
      mind though that it will still be present in the DOM so screen-readers can
      access it. Also, it is recommended not to change the label according to
      the state of the toggle
    </play-input-property-definition>
    <play-input-property-definition
      name="hideTexts"
      default="false"
      type="It takes a boolean">
      Whether or not you wish to display the label and the hint, if provided
    </play-input-property-definition>
    <play-input-property-definition
      name="checkedLabel"
      default="''"
      type="It takes a string">
      A string explaining what the toggle represents when it's checked
    </play-input-property-definition>
    <play-input-property-definition
      name="unCheckedLabel"
      default="''"
      type="It takes a string">
      A string explaining what the toggle represents when it's not checked
    </play-input-property-definition>
    <play-input-property-definition
      name="hint"
      type="It takes a string">
      It will display a description below the toggle. It is optional and can be
      hidden when <code>hideTexts</code> is <code>true</code>. You can use this
      as a way to make the purpose of the toggle more explicit than just with
      the label
    </play-input-property-definition>
    <play-input-property-definition
      name="align"
      default="right"
      type="It takes a string with only 2 possible values : <code>right</code> or <code>left</code>">
      Whether the label is on the left or right of the toggle
    </play-input-property-definition>
    <play-input-property-definition
      name="disabled"
      default="false"
      type="boolean">
      The value for the <code>disabled</code> attribute, for the underlying
      <code>input</code> tag
    </play-input-property-definition>
    <play-input-property-definition
      name="neutral"
      default="false"
      type="boolean">
      Whether or not the element will display like a toggle ( choose to activate
      something or not ) or like a switch ( choose between 2 equally legitimate
      possibilities ). When set to <code>true</code>, the toggle will not take
      any color when checked ( or in its activated state ) and the only visual
      clue that it has changed will be that its round part will have slid to the
      right
    </play-input-property-definition>
    <play-input-property-definition
      name="toggled()"
      type="Output">
      Emits the value of the toggle as boolean when the user toggles it
    </play-input-property-definition>
  </dl>
</section>

<section class="module-simulations">
  <h2>Play with the Component 😃</h2>
  <mat-card *ngIf="formToggle">
    <mat-card-header>
      <mat-card-title>Toggle element</mat-card-title>
    </mat-card-header>
    <mat-card-content class="simulation">
      <p>Example :</p>
      <pre>
  <span class="highlight-code">&lt;dsfr-toggle</span>
    toggleId="toggle-id"
    checkedLabel="checked"
    unCheckedLabel="unchecked"
    label="This is a label"
    hint="This is a hint"
    (toggled)="onToggled($event)"
  <span class="highlight-code">&gt;&lt;/dsfr-toggle&gt;</span>
</pre>
      <div class="settings">
        <form
          [formGroup]="formToggle"
          novalidate>
          <play-input-property-control
            formControlName="toggleId"
            name="toggleId"
            label="The toggle's Id"
            type="string">
          </play-input-property-control>
          <play-input-property-control
            formControlName="checkedLabel"
            name="checkedLabel"
            label="Toggle checked label"
            type="string">
          </play-input-property-control>
          <play-input-property-control
            formControlName="unCheckedLabel"
            name="unCheckedLabel"
            label="Toggle not checked label"
            type="string">
          </play-input-property-control>
          <play-input-property-control
            formControlName="label"
            name="label"
            label="Toggle's label"
            type="string">
          </play-input-property-control>
          <play-input-property-control
            formControlName="hint"
            name="hint"
            label="Toggle's hint"
            type="string">
          </play-input-property-control>
          <play-input-property-control
            formControlName="align"
            name="align"
            label="Toggle and label alignment"
            type="enum"
            [enum]="possibleAlignment">
          </play-input-property-control>
          <play-input-property-control
            formControlName="disabled"
            name="disabled"
            type="boolean">
          </play-input-property-control>
          <play-input-property-control
            formControlName="neutral"
            name="neutral"
            type="boolean">
          </play-input-property-control>
          <button
            mat-raised-button
            color="accent"
            (click)="formToggle.reset()">
            Reset
          </button>
        </form>
      </div>
      <div class="simulator-wrapper mat-elevation-z4">
        <play-simulator>
          <div *ngIf="form">
            <form [formGroup]="form">
              <dsfr-toggle
                formControlName="toggle"
                [toggleId]="formToggle.getRawValue().toggleId"
                [checkedLabel]="formToggle.getRawValue().checkedLabel"
                [unCheckedLabel]="formToggle.getRawValue().unCheckedLabel"
                [label]="formToggle.getRawValue().label"
                [hint]="formToggle.getRawValue().hint"
                [align]="formToggle.getRawValue().align"
                [disabled]="formToggle.getRawValue().disabled"
                [neutral]="formToggle.getRawValue().neutral">
              </dsfr-toggle>
            </form>
            <p>
              The toggle value is : <br />
              <strong>
                {{ form.value.toggle }}
              </strong>
            </p>
          </div>
        </play-simulator>
      </div>
    </mat-card-content>
  </mat-card>
</section>
```